src/Inciter/Partitioner.h

```diff
@@ -1260,6 +1265,7 @@ class Partitioner : public CBase_Partitioner< HostProxy,
                         CkCallback(CkReductionTarget(Transporter,load), host));
     }
     //! Compute average communication cost of merging the linear system
+    //! \param[in] host Host to signal to
```

Comment:
Are these computation tasks or wrappers for charm messages? Do we want to couple our compute so closely with our communication?